pkg/notes/document/template.go

```diff
@@ -63,6 +63,24 @@ filename | sha512 hash
 {{- end -}}
 # Changelog since {{$PreviousRevision}}
 
+{{with .CVEList -}}
+## Important Security Information
+
+This release contains changes that address the following vulnerabilities:
+{{range .}}
+### {{.Title}}
+
+{{.Description}}
+
+__Rating:__ {{.Rating}} — __Score:__ {{.Score}} — __Published:__ {{.Published}}
```

Comment:
What are Rating and Score? Is this referring to the severity bucket, and the numerical severity rating?
If so, we usually format those like:
> __Severity:__ Medium ([CVSS:3.1/AV:N/AC:H/PR:H/UI:R/S:U/C:H/I:H/A:H](https://www.first.org/cvss/calculator/3.1#CVSS:3.1/AV:N/AC:H/PR:H/UI:R/S:U/C:H/I:H/A:H))

If you don't want to include the full CVSS rating, you could just put the numeric score in parentheses.

Reply:
OK, I've modified the template to report following your suggestion. Added a new field for the vector too.